proposals/0176-high-level-interface-views-and-controllers.md

```diff
@@ -70,19 +70,11 @@ This method is called after `viewWillDisappear`.
 
 ### SDLViewController
 
-The view controller class is called `SDLViewController`. It implements the view controller and provides the necessary classes for the lifecycle. It's expected that the app developer creates subclasses of this base class.
+The view controller class is called `SDLViewController`. It's expected that the app developer creates subclasses of this base class. The methods for the lifecycle are realized using a protocol with optional methods. This way app developer can implement the methods in their subclasses without the need of calling super. `SDLViewController` implements the delegate but not the methods except `loadView`. The color scheme can be explicitly set presenting a view controller. When pushing another view controller to the stack it'll inherit the color scheme if not explicitly set.
```

Comment:
Please update the fourth sentence to "This way the app developer..." (adding "the").